snapcraft.yaml

```diff
@@ -1477,7 +1477,6 @@ parts:
       - bin/rsync
       - bin/setfacl
       - bin/sgdisk
-      - bin/unsquashfs
```

Comment:
Is this still getting built somewhere, where did it come from previously?

Reply:
```
$ ll /snap/core2?/current/usr/bin/unsquashfs 
-rwxr-xr-x 1 root root 114808 Oct 11  2021 /snap/core20/current/usr/bin/unsquashfs*
-rwxr-xr-x 1 root root 135064 Mar 25  2022 /snap/core22/current/usr/bin/unsquashfs*
```

Reply:
The staged package is `squashfs-tools` in the `lxd` part. The binary/package is also in the core bases:

```
$ grep squash /snap/core2?/current/usr/share/snappy/dpkg.list 
/snap/core20/current/usr/share/snappy/dpkg.list:ii  squashfs-tools                 1:4.4-1ubuntu0.3             amd64        Tool to create and append to squashfs filesystems
/snap/core22/current/usr/share/snappy/dpkg.list:ii  squashfs-tools                 1:4.5-3build1                           amd64        Tool to create and append to squashfs filesystems
```

I hope I got your question right this time around.

Reply:
Do we need the staged package anymore?